src/popup/components/Modals/MultisigProposalConfirmActions.vue

```diff
@@ -131,19 +135,20 @@ export default defineComponent({
       props.action === FUNCTION_TYPE_MULTISIG.confirm ? 'success' : 'critical'
     ));
 
-    const confirmActionContent = computed((): TranslateResult => {
+    const confirmActionContent = computed((): Record<string, TranslateResult> => {
       switch (props.action) {
         case FUNCTION_TYPE_MULTISIG.confirm:
-          return root.$t('pages.proposalDetails.signDialog');
+          return tm('pages.proposalDetails.signDialog');
         case FUNCTION_TYPE_MULTISIG.revoke:
-          return root.$t('pages.proposalDetails.revokeDialog');
+          return tm('pages.proposalDetails.revokeDialog');
         default:
-          return root.$t('pages.proposalDetails.refuseDialog');
+          return tm('pages.proposalDetails.refuseDialog');
       }
     });
 
     const actionHasError = computed(() => {
-      const confirmActionText = confirmActionContent.value as Record<string, TranslateResult>;
+      const confirmActionText = confirmActionContent.value as unknown as
+        Record<string, TranslateResult>;
```

Comment:
Is this needed here? The `confirmActionContent` already returns data tyoed this way.